src/types.ts

```diff
@@ -110,14 +111,14 @@ const asExchangeType = asValue('fiat', 'swap')
 
 export const asStandardTx = asObject({
   orderId: asString,
-  countryCode: asEither(asString, asNull),
+  countryCode: asEither(asString, asNull, asUndefined),
   depositTxid: asOptional(asString),
   depositAddress: asOptional(asString),
   depositCurrency: asString,
   depositAmount: asSafeNumber,
-  direction: asDirection,
-  exchangeType: asExchangeType,
-  paymentType: asEither(asFiatPaymentType, asNull),
+  direction: asOptional(asDirection),
+  exchangeType: asOptional(asExchangeType),
+  paymentType: asEither(asFiatPaymentType, asNull, asUndefined),
```

Comment:
What's the motivation for this change to the cleaner?

Reply:
These values don't exist in the current database, so validating the existing data from the database fails.